jobs/build/multi_promote/Jenkinsfile

```diff
@@ -129,11 +128,15 @@ node {
         currentBuild.displayName = release_name
     }
 
-    def arches = commonlib.ocpReleaseState["${major}.${minor}"]?.release
+    def major_minor = "${major}.${minor}"
+    def out = buildlib.doozer("--group=openshift-${major_minor} config:read-group --yaml release_state",
```

Comment:
also deprecated. better not touch

Reply:
Removed with https://github.com/openshift-eng/aos-cd-jobs/pull/3722